lib/active_model_serializers.rb

```diff
@@ -1,14 +1,16 @@
-require 'logger'
 require 'active_model'
 require 'active_support'
+require 'active_support/tagged_logging'
+require 'active_support/logger'
```

Comment:
So, for this I followed the pattern in active job

It's always nice to be explicit

You're right that it just so happens `require 'active_support'` requires logging and autoloads TaggedLogging, but there's no guarantee https://github.com/rails/rails/blob/4-0-stable/activesupport/lib/active_support.rb#L27